src/main/java/com/whiteclarkegroup/liquibaselinter/config/Config.java

```diff
@@ -29,18 +30,44 @@
     private final ListMultimap<String, RuleConfig> rules;
     private final boolean failFast;
     private final String enableAfter;
+    private final List<String> imports;
 
     @JsonCreator
     public Config(@JsonProperty("ignore-context-pattern") String ignoreContextPatternString,
                   @JsonProperty("ignore-files-pattern") String ignoreFilesPatternString,
                   @JsonProperty("rules") ListMultimap<String, RuleConfig> rules,
                   @JsonProperty("fail-fast") boolean failFast,
-                  @JsonProperty("enable-after") String enableAfter) {
-        this.ignoreContextPattern = ignoreContextPatternString != null ? Pattern.compile(ignoreContextPatternString) : null;
-        this.ignoreFilesPattern = ignoreFilesPatternString != null ? Pattern.compile(ignoreFilesPatternString) : null;
+                  @JsonProperty("enable-after") String enableAfter,
+                  @JsonProperty("import") @JsonFormat(with = JsonFormat.Feature.ACCEPT_SINGLE_VALUE_AS_ARRAY) List<String> imports) {
+        this(
+            ignoreContextPatternString != null ? Pattern.compile(ignoreContextPatternString) : null,
+            ignoreFilesPatternString != null ? Pattern.compile(ignoreFilesPatternString) : null,
+            rules,
+            failFast,
+            enableAfter,
+            imports);
+    }
+
+    public Config(Pattern ignoreContextPattern,
```

Comment:
Would be nice if we could reduce the need for all these constructors here. This one only seems to be needed in some tests so could just pass in the extra `imports` parameter for those. As for the below constructor we could just replace that with a method called `withRules` which rebuilds a new `Config` with the rules passed in. We can work around the `String` type for the patterns by calling `toString` on them, which will return their string representation.

Reply:
If you prefer the `withRules` approach, I'll switch this to use the builder pattern like `RuleConfig` does? That also allows me to handle the `Pattern` thing differently, and I believe more elegantly. It does make the tests a whole lot more readable using the Builder pattern.

Reply:
Builder pattern would also work nicely - go with that.